cEP-0010.md

```diff
@@ -136,10 +138,37 @@ Here is the list of aspects to be implemented along with their corresponding
 
 - `Root.Formatting`: This aspect checks on the format (the structure) of your
   codebase.
-- `Root.Formatting.Quotation`: This aspect checks on the quotations being used
-  in your files given `Root.Formatting.Quotation.preferred_quotation`.
+- `Root.Formatting.Quotation`: This aspect represents the quotation marks
+  being used in the files written in languages with multiple string delimiters,
+  which is common in recent languages, and excludes C-ish languages.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
-  user's preferred quotation.
+  preferred quotation marks for the selected files. It ensures that every string
+  contains the selected style of quotes.
+  For example, `'This is single quoted'` is changed to `"This is single quoted"`
+  if `preferred_quotation` is `"`.
+- `Root.Formatting.Quotation.escape_delimiter`: This taste ensures that
+  selected quotes are strictly followed even in cases where other quotation
+  marks are present inside string containing the selected style of quotes.
+  This is done by replacing other quotes with selected quotes and escaping
+  them wherever necessary.
+  For example, `"Hey 'coala'"` is changed to `"Hey \"coala\""` by setting
+  `preferred_quotation` as `"` and `escape_delimiter` as `True`.
+  Similarly, `'Hey "coala"'` is changed to `"Hey \"coala\""` with the
+  same settings as mentioned above.
+  For languages which do not support escaping quotes like `\"`, a warning
+  is shown. For such languages, taste `alternate_delimiter` is used
+  for quotation consistency.
+- `Root.Formatting.Quotation.alternate_delimiter`: This taste is used for
+  maintaining quotes consistency by using preferred quotes for files
+  written in languages which do not support escaping quotes like `\"` or `\'`.
+  For example, if the language is Fortran, `"Heya 'aspects'"` is converted to
+  `"Heya ""aspects"""` provided `alternate_delimiter=True` as Fortran supports
+  double quotes for escaping delimiters.
+  This functionality defaults to `False` as it is always preferred to use
```

Comment:
It should not default to `False`.  Maybe `None` is better.

For languages which use doubling up, it is the 'most' proper way to embed quote characters.

The alternative would be escape sequences, which are more ewww!

Note newer languages do not support doubling up, so this taste should be automatically disabled for those languages.